spark/src/main/scala/org/apache/comet/serde/structs.scala

```diff
@@ -167,3 +167,67 @@ object CometStructsToJson extends CometExpressionSerde[StructsToJson] {
     }
   }
 }
+
+object CometJsonToStructs extends CometExpressionSerde[JsonToStructs] {
+
+  override def getSupportLevel(expr: JsonToStructs): SupportLevel = {
+    // this feature is partially implemented and not comprehensively tested yet
```

Comment:
Do you have criteria for when this would change? You allude to arrays, maps, and fuzz tests in the PR description.